packages/turbo-gen/tsup.config.ts

```diff
@@ -1,6 +1,7 @@
 import { defineConfig, Options } from "tsup";
 import fs from "fs-extra";
-import chalk from "chalk";
+import { green } from "picocolors";
+import { hex } from "@turbo/utils/src/logger";
```

Comment:
This export doesn't exist. You'll want to re-export the `hex` function from `packages/turbo-utils/index.ts` to make it available at `@turbo/utils`.

Reply:
Thank you very much, I'll check that export and upload the change.